src/Simulation/TargetDefinitions/Interfaces/IGate_ApplyControlledX.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Quantum.Simulation.Core;
+
+namespace Microsoft.Quantum.Intrinsic.Interfaces
+{
+    public interface IGate_ApplyControlledX : IOperationFactory
+    {
+        Func<(Qubit, Qubit), QVoid> ApplyControlledX_Body();
```

Comment:
Before we ship these interfaces, it would be good to do the change to make the exposed methods in a more user friendly format (see my previous comment).

Reply:
Totally agreed. I'll make sure to adjust that as part of this PR.